src/features/preferences/renderer/preference-items/kubernetes/kubeconfig-sync/kubeconfig-sync.tsx

```diff
@@ -59,11 +59,7 @@ class NonInjectedKubeconfigSync extends React.Component<Dependencies> {
       reaction(
         () => Array.from(this.syncs.entries(), ([filePath, kind]) => tuple.from(filePath, kind)),
         syncs => {
-          action(() => {
-            for (const [path] of syncs) {
-              this.props.userStore.syncKubeconfigEntries.set(path, {});
```

Comment:
Was this empty object by accident? Was that the root cause? 

FYI: In the long run, we will be moving out from "saving happening in reaction" to mitigate this subset of bugs for good. The movement starts when first "Feature" wants to introduce new preferences (so very soon..) That will also make `UserStore` redundant. More on this topic when we have something to show.